src/util/anim_ease.ts

```diff
@@ -1,10 +1,18 @@
 import { twoMathPi, halfPI } from '../constants';
 
-type TEasingFunction = (
-  currentTime: number,
-  startValue: number,
-  byValue: number,
-  duration: number
+/**
+ * An easing function
+ * @param t current "time"/ms elapsed
+ * @param b start/"base" value
+ * @param c increment/"change"/"completion rate"/magnitude
+ * @param d "duration"
+ * @returns next value
+ */
+export type EasingFunction = (
+  t: number,
+  b: number,
+  c: number,
+  d: number
```

Comment:
this is a downgrade in clarity.
The function has 4 arguments, that for brevity years ago have been called t,b,c,d.
Now is extremely unclear what they do, and so in the type function those have been given a clear name.
Isn't this enough to have the clear name show in vscode? i would really leave the type as it is, and add the documentation on top if is necessary for some other tool

Reply:
Please restore this type as it was, both with the T and the names.

Reply:
I will restore the names, I started writing this a while ago and I didn't realize that the easing had been changed when I merged from master, derp my bad.
Still if the names change then I think it would be a good idea to change the easing function signatures because now they don't match anymore.

Reply:
Also I don't think clear names are enough, currentTime for instance could mean a lot of things or be in a lot different units. IMO specifying units in documentation is SUPER important

Reply:
Why not naming with units?
e.g. `durationMs`

Reply:
i m not against providing extra documentation, i just don't want to call those t,b,c,d.
Changing the easing function signatures could be done, in a separate PR or will bump up this in number of change and make hard to read what is changed.
Changes that are large and non significative for functionality should be done in isolation so that is easy to focus on changes.